tools/proto_format/proto_format.sh

```diff
@@ -23,9 +23,6 @@ read -ra BAZEL_BUILD_OPTIONS <<< "${BAZEL_BUILD_OPTIONS:-}"
 # Generate //versioning:active_protos.
 ./tools/proto_format/active_protos_gen.py ./api > ./api/versioning/BUILD
 
-# This is for local RBE setup, should be no-op for builds without RBE setting in bazelrc files.
-BAZEL_BUILD_OPTIONS+=("--remote_download_outputs=all")
```

Comment:
Did you verify with local RBE setup?

Reply:
i dont have that - but i did check the docs

Reply:
https://docs.bazel.build/versions/main/command-line-reference.html#flag--remote_download_outputs

Reply:
Bazel's default is irrelevant, this is to override what we specifies in .bazelrc: https://github.com/envoyproxy/envoy/blob/main/.bazelrc#L246

Reply:
my bad - ill revert and update docs